tests/test_api.py

```diff
@@ -0,0 +1,49 @@
+from lib import sdk
+
+import json, requests_mock
+
+s = sdk.SDK()
+
+token = "9a8b7c6d5e4f3g21"
+duration_params = {"type":2, "duration":10000}
+key_id = "123-456-789"
+url = "http://localhost"
+params = None
+
+
+def test_create(requests_mock):
+    requests_mock.register_uri("POST", url + "/keys", json=token, status_code=201)
+    r = s.keys.create(token, duration_params)
+    assert r.error.status == 0
+    assert token == r.value
+
+
```

Comment:
Kill extra blank line

Reply:
I have done so. But by flake8 standards, we need two extra lines
```bash
tests/test_api.py:20:1: E302 expected 2 blank lines, found 1
```